lib/steep/names.rb

```diff
@@ -24,7 +24,7 @@ def ==(other)
       end
 
       def hash
-        self.class.hash ^ name.hash ^ @absolute.hash
+        self.class.hash ^ name.hash ^ @location.hash
```

Comment:
Could you drop `@location.hash`? Because `@location` is not used in `==`, so we should not use the value in `hash` too.

Reply:
Sorry for late response. I remove the `@location.hash`, 

Should we add `^ @namespace.hash`? `@namespace` looks used in `#==`
https://github.com/soutaro/steep/blob/v0.8.2/lib/steep/names.rb#L22-L24

Reply:
Yes. Please add `namespce.hash`. I really did not notice that...

Reply:
I updated the commit to compute the hash from the same values used for `#==`.
Thank you for everything!